test/rules/no-mocha-global-arrows.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+var RuleTester = require('eslint').RuleTester,
+    rules = require('../../').rules,
+    ruleTester = new RuleTester(),
+    expectedErrorMessage = 'Do not pass arrow functions to it()';
+
+ruleTester.run('no-mocha-global-arrows', rules['no-mocha-global-arrows'], {
+
+    valid: [
+        'it()',
+        'it(function() { assert(something, false); })',
+        'it("should be false", function() { assert(something, false); })',
+        {
+            // In this example, `it` is not a global.
```

Comment:
Personally I also don’t have a need for this. So let’s rename the rule as you suggested, then we are safe for future extension of this rule 👍.